package.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "rancher-deskboard",
-  "version": "2.11.0",
+  "version": "2.11.1",
   "private": false,
   "engines": {
     "node": ">=20"
   },
+  "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72",
   "dependencies": {
-    "@rancher/shell": "3.0.2-rc.2"
+    "@rancher/shell": "3.0.5-rc.5"
```

Comment:
Do we want to keep relying on release candidates for the dashboard integration?

Reply:
Well, I tried 3.0.4 first, but that had issues opening the pod menu; as in, clicking ⋮ just did nothing.  The screenshot is from 3.0.5-rc.5.

![image](https://github.com/user-attachments/assets/5220de27-7a5b-4663-a60e-12e67628b1df)

Reply:
We could keep checking if `3.5.0` will be released before Rancher Desktop 1.20.0, and then bump it again, but I don't think it is necessary.

@rak-phillip I assume you will let us know if there is any significant fix upstream that we would want to include.